assets/sass/pages/_docs.scss

```diff
@@ -1151,25 +1151,54 @@
     .docs-wrap {}
 
     .texts-container {
-        h1, h5 {
-            margin-top: 5px;
-            padding-bottom: 0;
-
-            color: $c-blue-dark1;
-            font-family: 'Poppins';
+        h1 {
+            margin-bottom: 40px;
             font-size: 40px;
-            font-weight: 200;
+            margin-bottom: 20px;
```

Comment:
@catherineluse  `margin-bottom` is defined twice, the first will be ignored and should be removed

Reply:
deleted